auth.py

```diff
@@ -31,6 +31,14 @@ def is_site_admin(token):
     return False
 
 
+def get_refresh_token(token):
+    with open("/run/secrets/client_secret", 'r') as f:
```

Comment:
There isn't a client secret in the docker secrets anymore: this is now obtainable via `authx.auth.get_service_store_secret(service="keycloak", key="client-secret")`.